schemas_extra.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "fileMatch": ["*.sublime-build"],
+    "url": "sublime://schemas/sublime-build"
+  },
+  {
+    "fileMatch": ["*.sublime-color-scheme", "*.hidden-color-scheme"],
+    "url": "sublime://schemas/sublime-color-scheme"
+  },
+  {
+    "fileMatch": ["*.sublime-commands"],
+    "url": "sublime://schemas/sublime-commands"
+  },
+  {
+    "fileMatch": ["*.sublime-keymap"],
+    "url": "sublime://schemas/sublime-keymap"
+  },
+  {
+    "fileMatch": ["*.sublime-macro"],
+    "url": "sublime://schemas/sublime-macro"
```

Comment:
I really like this solution! It'll also allow us to make diagnostics more precise!